Octokit/Exceptions/RateLimitExceededException.cs

```diff
@@ -26,6 +26,7 @@ namespace Octokit
     public class RateLimitExceededException : ForbiddenException
     {
         readonly RateLimit _rateLimit;
+        private readonly TimeSpan _severTimeDiff = TimeSpan.Zero;
```

Comment:
:art: to be consistent with the other private member

```suggestion
        readonly TimeSpan _severTimeDiff = TimeSpan.Zero;
```